lib/impress.js

```diff
@@ -406,22 +400,21 @@ impress.server.stop = (
 };
 
 impress.shutdown = (code = 0) => {
-  if (!impress.finalization) {
-    impress.finalization = true;
-    if (process.isMaster) {
-      impress.killWorkers();
-      impress.log.server('Stopped server');
-      console.log('Impress shutting down'.green.bold);
-      impress.log.close(() => {
-        api.timers.setImmediate(() => {
-          impress.server.stop();
-          process.exit(code);
-        });
+  if (impress.finalization) return;
+  impress.finalization = true;
+  if (process.isMaster) {
+    impress.killWorkers();
+    impress.log.server('Stopped server');
+    console.log('Impress shutting down'.green.bold);
+    impress.log.close(() => {
+      api.timers.setImmediate(() => {
+        impress.server.stop();
```

Comment:
Didn't look into `impress.log.close`, just a question: is it guaranteed that logs are closed at this moment?